lib/api/filters/functions.php

```diff
@@ -45,6 +45,8 @@ function beans_add_filter( $hook, $callback_or_value, $priority = 10, $args = 1
  * Sub-hooks must be set in square brackets as part of the filter id argument. Sub-hooks are cascaded
  * in a similar way to CSS classes. Maximum 3 sub-hooks allowed.
  *
+ * Note: You can pass additional arguments to the functions that are hooked <tt>$id</tt>.
```

Comment:
Should this be `... functions that are hooked to ...` ?
I like moving the 'extra parameter' into the 'description' part.

Reply:
>I like moving the 'extra parameter' into the 'description' part.

Me too.  The more I thought about it, it seems like a better way to document this than in the `@param` listing.  We can now standardize it using this approach.